### ISSUE TITLE
[Feature] Compile a list of minecraft:seeds and add a basic list of seeds to grass drops

### ISSUE THREAD

**Issue by coldReactive** [EwyBoy/SeedDrop#10]
**Is your feature request related to a problem?**
No.

**Describe the solution you'd like**
Please consider finding a list of seeds currently able to be used in the world. Such as Cabbage Seeds from Farmer's Delight. However, certain mods, like Mystical Agriculture Seeds should be blacklisted; such as Nature Seeds.

**Describe alternatives you've considered**
I've tried asking the CurseForge discord for other mods that add random seed drops, and there isn't really any other mod.

**Additional context**
I've asked the modded minecraft discord: https://discord.com/channels/166630061217153024/418125698486632448/927907023360757771
And CurseForge discord about how to get a list of seeds: https://discord.com/channels/428228256236306434/768713291643486218/927906067772497960

***

Basically, they suggested to add your own tag list, such as `seedrop:seeds_dropped_by_grass` and set it to grab everything from `minecraft:seeds` and `forge:seeds`.

**Comment by EwyBoy** [comment]
Can be done, but want save you much trouble as I doubt most modders add a seedtag to their seeds.
